altair/utils/save.py

```diff
@@ -130,6 +128,8 @@ def save(
     """
     if json_kwds is None:
         json_kwds = {}
+    fp = pathlib.Path(fp) if isinstance(fp, str) else fp
```

Comment:
Having `encoding` up here makes sense to me, thanks! As I'd suggest to add back the functionality for `str` in the other functions, the other line here with the conversion to a Path is no longer necessary.